doc/app-example/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+CMAKE_MINIMUM_REQUIRED(VERSION 2.8)
+
+PROJECT(st-hello)
+SET(PROJECT_VERSION   0.1)
+SET(SRCS main.c)
+
+FIND_PACKAGE(PkgConfig)
```

Comment:
This only works for systems where `pkg-config` is present. It will not work on vanilla windows/osx systems.